src/sidebar.js

```diff
@@ -41,21 +41,17 @@ const activityTab = new OCA.Files.Sidebar.Tab({
 	iconSvg: LightningBolt,
 
 	async mount(el, fileInfo, context) {
-		// only load if needed
-		if (ActivityTabView === null) {
-			const { default: ActivityTab } = await import('./views/ActivityTab.vue')
-			ActivityTabView = ActivityTabView ?? Vue.extend(ActivityTab)
-		}
-		// destroy previous instance if available
+		const { default: ActivityTab } = await import(/* webpackPreload: true */ './views/ActivityTab.vue')
+		ActivityTabView = ActivityTabView ?? Vue.extend(ActivityTab)
```

Comment:
Won't this revert this optimisation?
https://github.com/nextcloud/activity/pull/1363